### PR TITLE
Add example targets

### PR DESCRIPTION
**Pull request by zaharidichev**
Makes things easier to run.

Signed-off-by: Zahari Dichev <zaharidichev@gmail.com>

### CODE REVIEW

File: console-subscriber/Cargo.toml [hunk 1]
```diff
@@ -22,4 +22,12 @@ futures = { version = "0.3", default-features = false }
...
\ No newline at end of file
+tracing-subscriber = { version = "0.2.17", features = ["fmt", "registry", "env-filter"] }
+
+[[example]]
+name = "app"
+path = "examples/app.rs"
+
+[[example]]
+name = "dump"
+path = "examples/dump.rs"
```
Comment (hawkw) [inline console-subscriber/Cargo.toml]: are these necessary for the examples to work? i know i've used `cargo run --example` to run them locally without having to add this to the Cargo.toml...

File: README.md [hunk 2]
```diff
@@ -126,6 +126,12 @@ tools**:
 * `dump.rs`: a simple CLI program that dumps the data stream from a `Tasks`
   server
 
+Examples can be executed with:
+
+```
```
Comment (hawkw) [inline README.md]: nit:
```suggestion
```shell
```

**Review by hawkw — APPROVED** [review]

File: console-subscriber/Cargo.toml (hunk 1 above)
Comment (seanmonstar) [reply console-subscriber/Cargo.toml]: Same, I also run (from the `console-subscriber` directory) `cargo run --example app` and it works...

Comment (hawkw) [reply console-subscriber/Cargo.toml]: > Same, I also run (from the `console-subscriber` directory) `cargo run --example app` and it works...

you should be able to do it even from the repo root, at least on Rust 1.52.0 --- i've been doing this

Comment (zaharidichev) [reply console-subscriber/Cargo.toml]: yes in deed, this is the case. Will update

**Pull request merged**